### PR TITLE
First TOF commit

### PR DESCRIPTION
**Pull request by noferini**
I added just the TOF dir

### CODE REVIEW

File: Detectors/TOF/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Start on 13th April 2017
```
Comment (matthiasrichter) [inline Detectors/TOF/README.md]: IMHO this single commit does not justify a PR, I would postpone it until there is content.

**Comment by sawenzel** [comment]
@noferini : Adding to @matthiasrichter comment: Please try to avoid "merge commits". The preferred workflow is that you start a new development fork or branch based on the dev branch ... and then regularly rebase your branch on top of developments in the original dev branch.
 
So it is:

`git rebase origin/dev`
instead of
`git merge origin/dev`

If the rebase fails ... you can contact me for further advice.

**Comment by noferini** [comment]
Dear all,
what you said is fine for me.
It was just a test and I would like to know if everything was correct.

**Comment by sawenzel** [comment]
testing the workflow is fine and necessary. You see that your change compiled fine ... closing this now.

**Pull request closed without merging**